wasm.cmake

```diff
@@ -375,3 +375,4 @@ endif()
 # Force ABI for Hunter to work
 set (CMAKE_C_ABI_COMPILED emcc)
 set (CMAKE_CXX_ABI_COMPILED emcxx)
+set (CMAKE_BUILD_WITH_INSTALL_RPATH ON CACHE BOOL "Force CMAKE_BUILD_WITH_INSTALL_RPATH" FORCE )
```

Comment:
Can you elaborate on why on wasm we need Runtime path now ? That's pretty unexpected.

Reply:
26: -- Configuring done
2023-08-09T20:21:01.4462174Z 26: CMake Error at CMakeLists.txt:62 (add_executable):
2023-08-09T20:21:01.4462576Z 26:   The install of the app target requires changing an RPATH from the build
2023-08-09T20:21:01.4463006Z 26:   tree, but this is not supported with the Ninja generator unless on an
2023-08-09T20:21:01.4463540Z 26:   ELF-based or XCOFF-based platform.  The CMAKE_BUILD_WITH_INSTALL_RPATH
2023-08-09T20:21:01.4463952Z 26:   variable may be set to avoid this relinking step.
2023-08-09T20:21:01.4464234Z 26: 
2023-08-09T20:21:01.4464442Z 26: 
2023-08-09T20:21:01.4464724Z 26: CMake Error at CMakeLists.txt:62 (add_executable):
2023-08-09T20:21:01.4465163Z 26:   The install of the app target requires changing an RPATH from the build
2023-08-09T20:21:01.4465747Z 26:   tree, but this is not supported with the Ninja generator unless on an
2023-08-09T20:21:01.4466192Z 26:   ELF-based or XCOFF-based platform.  The CMAKE_BUILD_WITH_INSTALL_RPATH
2023-08-09T20:21:01.4466531Z 26:   variable may be set to avoid this relinking step.
2023-08-09T20:21:01.4466776Z 26: 
2023-08-09T20:21:01.4467002Z 26: 
2023-08-09T20:21:01.4467236Z 26: -- Generating done
2023-08-09T20:21:01.4467614Z 26: CMake Generate step failed.  Build files cannot be regenerated correctly.
2023-08-09T20:21:01.4468348Z 26: ● [err] Dynamic exception type: std::runtime_error
2023-08-09T20:21:01.4468873Z 26: std::exception::what: configure failed in /usr/local/share/.tipi/vD.w/3176b36-example-test-defines.b/08fea58/bin
2023-08-09T20:21:01.4469623Z 26: ● [err]

Because if with wasm we have some define I have this kind of error before adding this cmake option

Reply:
Well I'm unsure, but in my remembering wasm in emscripten is ELF based. I can be wrong though.